app/views/applications/edit.html.haml

```diff
@@ -124,8 +124,8 @@
       Do you need a scholarship ticket to be able to attend?
     %span (Required)
     %p.small
-      All tickets include lodging, food, and transportation to the venue from downtown San Francisco.
-      Although official programming will conclude Sunday night, attendees will be welcome to stay until
+      Tickets cost $325. All tickets include lodging, food, and transportation to the venue from downtown San Francisco.
+      Official conference activities will conclude Sunday night, but attendees are welcome to stay until
```

Comment:
The word "programming" confused me here for a second ('cause programming conference) so I did some adjusting.